lib/src/solid/login.dart

```diff
@@ -573,3 +606,69 @@ class _SolidLoginState extends State<SolidLogin> {
     );
   }
 }
+
+class PodButton extends StatelessWidget {
```

Comment:
How about the PodButton constructor accepts an instance of PodButtonStyle, this is likely simply the SolidLogin constructor?